rbac/rbac/middleware.py

```diff
@@ -196,30 +195,6 @@ def process_response(self, request, response):  # pylint: disable=no-self-use
                 is_admin = is_system = False
                 account = None
 
-        extras = {}
-
-        if "ecs" in settings.LOGGING_HANDLERS:
-            extras = {
-                "http": {
-                    "request": {
-                        "body": {"bytes": sys.getsizeof(request.body)},
-                        "bytes": sys.getsizeof(request),
-                        "method": request.method,
-                    },
-                    "response": {
-                        "body": {"bytes": sys.getsizeof(response.content)},
-                        "bytes": sys.getsizeof(response),
-                        "status_code": response.status_code,
-                    },
-                },
-                "url": {
-                    "original": request.path + query_string,
-                    "path": request.path,
-                    "query": query_string,
-                    "port": request.get_port(),
-                },
-            }
```

Comment:
Pulling this out leaves us lacking a lot of data we'll want for the ECS logging format, may be worth creating a future ticket to get that additional data added back in, working around the WSGIRequest and socket object problems.